uml/delete_message.plantuml

```diff
@@ -0,0 +1,16 @@
+@startuml Delete messages
```

Comment:
Really nice diagrams above - very clear and neat.

I haven't seen the type of files below before so I'm unsure what they do!